### PR TITLE
D8CORE-2105 Upgrade decanter to 6.2.0

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Upgrade decanter

# Need Review By (Date)
- 12/3

# Urgency
- medium

# Steps to Test
1. view the file changes.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: dist/templates/decanter/components/media/media.twig [hunk 1]
```diff
@@ -28,6 +28,12 @@
  * - media_caption: Optional caption for the media.
  */
 #}
+
+{# Set link to null if there is no content to prevent an empty link #}
```
Comment (jenbreese) [inline dist/templates/decanter/components/media/media.twig]: @pookmish this doesn't seem to work correctly. The other parts of the pull request  do work properly.  What would be the steps to get this one fixed? I have a ticket already for this, D8CORE-2529 which is what this below was supposed to fix.

Comment (pookmish) [reply dist/templates/decanter/components/media/media.twig]: I don't know. Due to it's inheritance from decanter I avoid it. Decanter never plays nice for the things I want it to and I constantly fight it.

Comment (jenbreese) [reply dist/templates/decanter/components/media/media.twig]: I have a fix for the media problem. I'll put that together as a decanter pull request and then that would fix this problem.

Comment (sherakama) [reply dist/templates/decanter/components/media/media.twig]: > Decanter never plays nice for the things I want it to and I constantly fight it.

You know, you do have the ability to fix the things you don't like or at the very least, report them and create issue tickets.

Comment (pookmish) [reply dist/templates/decanter/components/media/media.twig]: The spider web that is Decanter makes it very difficult for me to navigate and make any changes

**Comment by jenbreese** [comment]
@pookmish The fix to decanter has been done. I think if you pull that in then I can approve this.

**Comment by pookmish** [comment]
I believe Decanter would need to get a new release punched in order to bring in those changes.

**Comment by sherakama** [comment]
> I believe Decanter would need to get a new release punched in order to bring in those changes.

I can hook you up with a point release.

**Comment by sherakama** [comment]
https://github.com/SU-SWS/decanter/releases/tag/6.2.1

**Comment by pookmish** [comment]
@jenbreese decanter has been updated [`786c453` (#82)](https://github.com/SU-SWS/jumpstart_ui/pull/82/commits/786c453ad4f5214a9559faa83ae3c661fefec25c#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R21)

**Review by jenbreese — APPROVED** [review]
Looks good. Thank you

**Pull request merged**